go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/yudai/gojsondiff v1.0.0
```

Comment:
Can you please undo changes made to go.mod and go.sum?

Reply:
@Sonali-More-Xandr Got it. It changes perhaps because I did the local end to end test with the command `go build .`

Reply:
DONE